galaxy.yml.j2

```diff
@@ -35,3 +35,5 @@ tags:
     - tower_utils
 dependencies:
     "ansible.posix": ">=1.0.0"
+    "kubernetes.core": ">=2.2.0"
```

Comment:
The collection requirements and the role meta collections are different, I believe these are all 4 collections correct?

collections:
  - redhat.openshift
  - community.okd
  
  Just checking that we have the right dependencies declared.

Reply:
community.okd is the upstream version of redhat.openshift. I chose to depend on redhat.openshift for the collection as that is the Red Hat supported collection. The role however will use either community.okd or redhat.openshift for the tasks. 

I'm thinking about opening an RFE for collection dependencies to introduce support for depending on one or another collection and have a preference for one over the other. There are several collections where this would be useful (awx.awx and ansible.controller is another example).